integration/nwo/fabric/network/network_support.go

```diff
@@ -1600,7 +1600,8 @@ func (n *Network) Chaincodes(channel string) []*topology.ChannelChaincode {
 	return res
 }
 
-// BCCSPOpts return the BCCSP configuration with the passed default provider
+// BCCSPOpts returns a `topology.BCCSP` instance. `defaultProvider` sets the `Default` value of the BCCSP,
```

Comment:
the param is now called `provider` and not `defaultProvider` :D

Reply:
fixed 😅